### PR TITLE
WIP: add layout component

### PR DESCRIPTION
**Pull request by YoshioHara**
resubmit
https://github.com/drecom/scene-graph-mediator-cli/pull/6
https://github.com/drecom/scene-graph-mediator-rt/pull/12

### CODE REVIEW

File: packages/scene-graph-mediator/runtime/test/importer/Pixi.js [hunk 1]
```diff
@@ -452,5 +461,356 @@ describe('Pixi', () => {
...
+        metadata: metadata
+      };
+
+      describe('when layout component is given', () => {
+        
+        before(async()=>{
+          //  FIXME:
+          //  Measures to avoid the problem that sprite size becomes default size (1, 1).
+          //  want to delete if there is another solution.
+          const root = pixi.import(schema);
```
Comment (YoshioHara) [inline packages/scene-graph-mediator/runtime/test/importer/Pixi.js]: Correspondence for the problem that sprite size at the time of test becomes default value (1, 1).
want to respond if there is an appropriate correction method.

File: packages/scene-graph-mediator/runtime/src/importer/Pixi.ts [hunk 2]
```diff
@@ -294,13 +295,13 @@ export default class Pixi extends Importer {
...
     } else if (node.sprite) {
-      // TODO: base64 image
-
       let texture = null;
       if (node.sprite.atlasUrl && node.sprite.frameName) {
         texture = PIXI.Texture.fromFrame(node.sprite.frameName);
       } else if (node.sprite.url) {
         texture = resources[node.sprite.url].texture;
+      } else if (node.sprite.base64) {
+        texture = PIXI.Texture.fromImage(node.sprite.base64);
```
Comment (YoshioHara) [inline packages/scene-graph-mediator/runtime/src/importer/Pixi.ts]: Supports base64 texture to make it easy to use texture for testing.

Comment (dolow) [reply packages/scene-graph-mediator/runtime/src/importer/Pixi.ts]: thx!

File: packages/scene-graph-mediator/runtime/test/importer/Pixi.js [hunk 3]
```diff
@@ -17,6 +18,14 @@ function clearCache() {
   PIXI.BaseTexture.removeFromCache(testSpriteUrl);
 }
 
+function sleep(msec) {
```
Comment (dolow) [inline packages/scene-graph-mediator/runtime/test/importer/Pixi.js]: it seems unused.

File: packages/scene-graph-mediator/runtime/test/importer/Pixi.js (hunk 1 above)
Comment (dolow) [reply packages/scene-graph-mediator/runtime/test/importer/Pixi.js]: ok, let's fix this in another PR.

File: packages/scene-graph-mediator/runtime/test/importer/Pixi.js (hunk 3 above)
Comment (YoshioHara) [reply packages/scene-graph-mediator/runtime/test/importer/Pixi.js]: deleted unused methods.
https://github.com/drecom/scene-graph/pull/28/commits/531faf869cbe5c519477af9d9482ec60713fb46e

**Pull request merged**